src/slides/react-state/react-state.js

```diff
@@ -11,10 +11,9 @@ import * as stateExample4 from "!raw-loader!./state-4.example";
 
 const Slide1 = ListSlideMaker(
   `React State`,
-  [`While React Props are available in both function components and class components, 'state' can only exist on a class component`,
-  `In class components, you have the opportunity to control when and how your component re-renders`,
-  `This combines with the ability to store state in your component (which you cannot do in function components, aka, stateless components)`,
-  `What this means is that there is a keyword property, 'state', that you can interact with in your class component`]
+  [`While React Props are available in both function components and class components, class components can also hold 'state'`,
```

Comment:
Should we mention Function Components can hold state via Hooks here?

Reply:
I updated the copy a bit here to clarify that both types of components can have state, but that the API described in these slides is specifically for class components. Thanks for the feedback!

Reply:
(I didn't want to get into the nitty gritty of how function components use state since that's covered thoroughly in the hooks section later, so I left it open-ended.)